unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -4043,17 +4043,17 @@ public void SetObjectPoses(ServerAction action)
                 actionFinished(false);
                 return;
             }
-            StartCoroutine(setObjectPoses(action.objectPoses));
+            StartCoroutine(setObjectPoses(action.objectPoses, action.placeStationary));
         }
 
         // SetObjectPoses is performed in a coroutine otherwise if
         // a frame does not pass prior to this AND the imageSynthesis
         // is enabled for say depth or normals, Unity will crash on 
         // a subsequent scene reset()
-        protected IEnumerator setObjectPoses(ObjectPose[] objectPoses){
+        protected IEnumerator setObjectPoses(ObjectPose[] objectPoses, bool placeStationary){
             yield return new WaitForEndOfFrame();
-            bool success = physicsSceneManager.SetObjectPoses(objectPoses);
-            actionFinished(success);
+            bool success = physicsSceneManager.SetObjectPoses(objectPoses, errorMessage, placeStationary);
```

Comment:
Add `out` keyword to errorMessage (assuming you want the value updated).

Reply:
yes, added